tests/test_10_use_cases.py

```diff
@@ -0,0 +1,210 @@
+import json
+import re
+import time
+from datetime import datetime, timedelta
+import uuid
+
+import pytest
+from random import randrange
+import requests
+
+from tests.helpers import NetworkTest
+
+SDX_CONTROLLER = 'http://sdx-controller:8080/SDX-Controller'
+KYTOS_TOPO_API = "http://%s:8181/api/kytos/topology/v3"
+KYTOS_SDX_API  = "http://%s:8181/api/kytos/sdx"
+KYTOS_API = 'http://%s:8181/api/kytos'
+
+class TestUseCases:
+    net = None
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(["ampath", "sax", "tenet"])
+        cls.net.wait_switches_connect()
+        cls.net.run_setup_topo()
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    @classmethod
+    def setup_method(cls):
+        api_url = SDX_CONTROLLER + '/l2vpn/1.0'
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+        response_json = response.json()
+        for l2vpn in response_json:
+            response = requests.delete(api_url+f'/{l2vpn}')
+
+    def test_010_send_topo_update_with_intra_domain_link_down_path_found(self):
```

Comment:
I believe the name of the method is inverted with the next one. This test 10 would be the "path not found" (since you disabled the link Tenet01 -- Tenet03, there will be no alternative paths)